src/shared/components/unsupportedBrowserModal/UnsupportedBrowserModal.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
 import { Modal } from 'react-bootstrap';
-import browser from 'detect-browser';
+const browser = require('bowser');
```

Comment:
@Tamba32 still missing bowser dependency, right? Surprised tests work fine, is it part of one of the other packages?